From 8ee93c8765efa94775c147669e559c81a9c8e8a7 Mon Sep 17 00:00:00 2001
From: Mete Alpaslan <mete@pardus.org.tr>
Date: Wed, 1 Dec 2010 21:32:28 +0200
Subject: [PATCH 2/4] yali.storage.formats.filesystems:Filesystem modules must be loaded

If any filesystem type are not found in kernel filesystems list which is
created from /proc/filesystem, backend would not estimate format type is
mountable or not.

Every supported storage format's kernel module must be loaded.
---
 yali/storage/formats/filesystem.py |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/yali/storage/formats/filesystem.py b/yali/storage/formats/filesystem.py
index 7ab000a..5d37984 100644
--- a/yali/storage/formats/filesystem.py
+++ b/yali/storage/formats/filesystem.py
@@ -97,6 +97,9 @@ class Filesystem(Format):
 
         self._targetSize = self._size
 
+        if self.supported:
+            self.loadModule()
+
     def __str__(self):
         s = Format.__str__(self)
         s += ("  mountpoint = %(mountpoint)s  mountopts = %(mountopts)s\n"
-- 
1.7.2.2

